.blackfire.yml

```diff
@@ -1,3 +1,6 @@
+# Copyright © Magento, Inc. All rights reserved.
```

Comment:
Not magento file - I guess we don't need copyright here.

Reply:
Fixed